src/main/java/com/nomiceu/nomilabs/mixin/vanilla/DedicatedServerMixin.java

```diff
@@ -69,15 +66,15 @@ public void motdSubstitutions(DedicatedServer instance, String s) {
 
         NomiLabs.LOGGER.info("Enabling Labs MOTD Substitutions...");
         instance.setMOTD(s.replace("{version}", LabsVersionConfig.formattedVersion)
-                .replace("{mode}", LabsModeHelper.getFormattedMode()));
+                .replace("{mode}", "Hog"));
```

Comment:
To my understanding this just makes it so that {mode} is replaced by Hog in server MOTDs. IIRC the {mode} was removed from the default MOTD in Hogifactory, so i'm not sure it still needs to exist here.